horizons/__init__.py

```diff
@@ -0,0 +1,260 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+import dateutil.parser
+from common.minimodel import Target, TargetTag, Site
+from common.helpers.helpers import dms2rad, hms2rad, angular_distance
+import numpy as np
+import numpy.typing as npt
+import requests
+from typing import Tuple, Union
+import logging
+import contextlib
+import os
+
+@dataclass
+class EphemerisCoordinates:
+    """
+    Both ra and dec are in radians.
```

Comment:
It looks like we're going to be working with degrees, I believe, so we will have to do a small conversion here.